sdk/driver/driver_test.go

```diff
@@ -0,0 +1,136 @@
+package driver
+
+import (
+	"testing"
+
+	"gopkg.in/bblfsh/sdk.v1/protocol"
+	"gopkg.in/bblfsh/sdk.v1/uast"
+
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	ManifestLocation = "internal/native/manifest.toml"
+}
+
+func TestDriverParserParse(t *testing.T) {
+	require := require.New(t)
+	NativeBinary = "internal/native/mock"
+
+	d, err := NewDriver(&uast.ObjectToNode{}, nil)
+	require.NoError(err)
+	require.NotNil(d)
+
+	err = d.Start()
+	require.NoError(err)
+
+	r := d.Parse(&protocol.ParseRequest{
+		Language: "fixture",
+		Content:  "foo",
+	})
+
+	require.NotNil(r)
+	require.Equal(len(r.Errors), 0)
+	require.Equal(r.Status, protocol.Ok)
+	require.Equal(r.Elapsed.Nanoseconds() > 0, true)
+	require.Equal(r.UAST.String(), " "+
+		"{\n"+
+		".  Roles: Unannotated\n"+
+		".  Properties: {\n"+
+		".  .  key: val\n"+
+		".  }\n"+
+		"}\n",
+	)
+
+	err = d.Stop()
+	require.NoError(err)
+}
+
+func TestDriverParserParse_MissingLanguage(t *testing.T) {
+	require := require.New(t)
+
+	d, err := NewDriver(&uast.ObjectToNode{}, nil)
+	require.NoError(err)
+	require.NotNil(d)
+
+	err = d.Start()
+	require.NoError(err)
+
+	r := d.Parse(&protocol.ParseRequest{
+		Content: "foo",
+	})
+
+	require.NotNil(r)
+	require.Equal(len(r.Errors), 1)
+	require.Equal(r.Status, protocol.Fatal)
+	require.Equal(r.Elapsed.Nanoseconds() > 0, true)
+	require.Nil(r.UAST)
+
+	err = d.Stop()
+	require.NoError(err)
+}
+func TestDriverParserParse_Malfunctioning(t *testing.T) {
+	require := require.New(t)
+	NativeBinary = "echo"
+
+	d, err := NewDriver(&uast.ObjectToNode{}, nil)
+	require.NoError(err)
+	require.NotNil(d)
+
+	err = d.Start()
+	require.NoError(err)
+
+	r := d.Parse(&protocol.ParseRequest{
+		Language: "fixture",
+		Content:  "foo",
+	})
+
```

Comment:
Nitpicking: Inconsistent newline (other tests have the NoError/NotNil test just below the calls/struct literals).